### PR TITLE
Add pimsviewer feedstock

### PR DESCRIPTION
**Pull request by rbnvrw**
Add a feedstock for pimsviewer, a GUI for pims.

### CODE REVIEW

File: recipes/pimsviewer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  license: GPL-3.0
+  license_file: LICENSE
+  summary: 'A graphical user interface (GUI) for PIMS'
+
+  description: A graphical user interface (GUI) for PIMS
+
+extra:
+  recipe-maintainers:
+    - rbnvrw
+    - caspervdw
```
Comment (jakirkham) [inline recipes/pimsviewer/meta.yaml]: Are you ok being a maintainer here, @caspervdw?

File: recipes/pimsviewer/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  imports:
+    - pimsviewer
+
+about:
+  home: https://github.com/soft-matter/pimsviewer
+  license: GPL-3.0
+  license_file: LICENSE
+  summary: 'A graphical user interface (GUI) for PIMS'
+
+  description: A graphical user interface (GUI) for PIMS
```
Comment (jakirkham) [inline recipes/pimsviewer/meta.yaml]: Would drop the blank line in-between `summary` and `description`.

**Comment by jakirkham** [comment]
cc @conda-forge/pims

**Comment by rbnvrw** [comment]
Thank you for your feedback @jakirkham , I have implemented the changes.

File: recipes/pimsviewer/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```
Comment (jakirkham) [inline recipes/pimsviewer/meta.yaml]: Would it make sense to use [`noarch: python`]( https://conda-forge.org/docs/meta.html#building-noarch-packages ) here?

Comment (rbnvrw) [reply recipes/pimsviewer/meta.yaml]: Yes, I think it is possible, I overlooked this option. Thank you!

**Comment by rbnvrw** [comment]
Is this good to merge @jakirkham ?

**Comment by jakirkham** [comment]
Waiting for the other maintainer to sign off.

**Comment by jakirkham** [comment]
Friendly nudge, @caspervdw. 😉

**Comment by rbnvrw** [comment]
I think @caspervdw  does not have time to be a maintainer at the moment, I have removed him for now.

**Pull request merged**